src/main/java/com/globalitgeeks/examninja/usermanagement/service/UserService.java

```diff
@@ -56,6 +56,20 @@ private boolean isValidPassword(String password) {
     }
 
     // Login user
+    public User login(String email, String password) throws Exception {
```

Comment:
User ResponseEntity to return the response instead of returning User

Reply:
Addressed